mycroft/configuration/mycroft.ini

```diff
@@ -93,3 +93,10 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[PhillipsHueSkill]
+verbose = False
```

Comment:
Ok, cool. If I do lose via the democratic process here though, can I ask if there's a practical reason to remove it? The overhead for keeping it seems pretty negligible (it's one extra line per intent type that makes use of it, and an essentially non-existent perf hit from the extra ifs) so I'm curious what your concerns are.

Just as a note, this functionality is important enough to me as a user, I would remove this if I had to to get it into core, but fork and add it back for my own Mycroft. I really, really hate the confirmations from the Echo, and wish I could disable them there.